src/components/LeftSidebar/LeftSidebar.vue

```diff
@@ -644,10 +731,25 @@ export default {
 
 .conversations-search {
 	flex-grow: 1;
+	transition: all 0.7s ease;
+	width: 65%;
+	position : absolute;
+	left: 8px; //padding
```

Comment:
```suggestion
	padding: 0 8px;
```
and with that we may use `width: 100%` in expanded mode